### ISSUE TITLE
The result becomes 0 when the rate high

### ISSUE THREAD

**Issue by lemorn** [robertdavidgraham/masscan#548]
Hello, I found an interesting thing. I use Amazon cloud server with 5G bandwidth. When I change the rate parameter, the scan result will suddenly change from the normal value to 0. Why does it suddenly become 0 instead of less?Thank you！
![image](https://user-images.githubusercontent.com/53805865/105819342-04f83880-5ff3-11eb-9286-56fd2487ae94.png)

**Comment by lemorn** [comment]
The same problem.
Today I used a shell script to scan several network segments continuously, and the parameters were set as follows, 
![image](https://user-images.githubusercontent.com/53805865/106014643-a1583300-60f8-11eb-8568-e9440ce06450.png)
and the results obtained showed that the scan results of several network segments were 0. 
![image](https://user-images.githubusercontent.com/53805865/106014530-884f8200-60f8-11eb-88b6-bbb55193f2b9.png)
After capturing packets with tcpdump and analyzing with wireshark, it was found that after the first scan, the server did not receive any packets for nearly 25 seconds. 
![image](https://user-images.githubusercontent.com/53805865/106014768-c187f200-60f8-11eb-88bd-a59ff2871b19.png)

Then I adjusted the wait parameter to 30, and the result was greatly improved. 
![image](https://user-images.githubusercontent.com/53805865/106016500-7d95ec80-60fa-11eb-852e-2f410eb76018.png)

I want to know why this is? Is it because my server is restricted? I am using Amazon's EC2 server.
Thanks.
